src/components/atoms/sidebar-item.tsx

```diff
@@ -0,0 +1,42 @@
+import { useHash } from 'react-use';
+
+import type { Section } from '@/types/data';
+import type { IconName } from '@/types/icon';
+
+import Icon from './icon';
+
+export interface Props {
+  section: Section;
+  icon: IconName;
+}
+
+const MAIN_SECTION: Section = 'main';
+
+const SidebarItem = ({ section, icon }: Props) => {
+  const [hash] = useHash();
+  const href = `#${section}`;
+
+  const isActive = () => {
```

Comment:
Można to w sumie skrócić do ternary podczas przypisania do zmiennej i wtedy pozbyć się funkcji